src/components/LocationListener.tsx

```diff
@@ -72,13 +72,16 @@ class LocationListener extends Component<Props> {
     const {
       coords: {latitude, longitude},
       timestamp,
+      speed,
```

Comment:
As far as I can tell there is no `speed` property coming from `Location`. Am I missing something?

Reply:
I think it's an object within `coords` so if you move it there it should be fine

Reply:
Good catches. Thank you